gel-dsn/src/gel/params.rs

```diff
@@ -1180,8 +1180,8 @@ fn parse_instance(local: &str, context: &impl BuildContext) -> Result<Params, Pa
 fn parse_cloud(profile: &str, context: &impl BuildContext) -> Result<Params, ParseError> {
     let mut explicit = Params::default();
 
-    let Some(cloud_credentials): Option<CloudCredentialsFile> =
-        context.read_config_file(format!("cloud-credentials/{profile}.json"))?
```

Comment:
Hmm, I wonder if we do this somewhere else.

Reply:
I believe both slashes work in windows but the paths become a large mess. We definitely have a lot of this style of path all over